GoogleSignIn/Sources/GIDProfileDataFetcher/API/GIDProfileDataFetcher.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+@class OIDAuthState;
+@class GIDProfileData;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol GIDProfileDataFetcher <NSObject>
+
+/// Fetches the latest @GIDProfileData object.
```

Comment:
Would be nice to provide more detail here about what this method does at a high level.